### DISCUSSION TITLE
Linear Heat Detection in PyroSim

### DISCUSSION THREAD

**Discussion by camborst** [firemodels/fds#13162] [Q&A]
Is there a way to model linear heat detectors in pyrosim? Or any alternatives?

Thanks,
-Cam

**Comment by drjfloyd** [comment]
General construction is a wire consisting of a twisted pair of conductors seperated by a low melting point membrane wrapped in a protective jacket. When the membrane melts, the conductors short.  You could

-monitor the gas temperature along the LHD path and consider detection to be whenever the gas temperature exceeds the LHD temperature rating. This will of course ignore the delay due to the need transfer heat into the cable
-define a particle class representing the cable construction. place a series of those particles along the path. monitor the temperature of each particle beneath the outer jacket and consider detection to be when that temperature exceeds the LHD temperature rating.
